src/parser/grammar.mly

```diff
@@ -348,7 +361,7 @@ clause:
 
 attributes:
 | { [] }
-| EXTERNAL { [ External ] }
+| EXTERNAL; o = option(STRING) { [ External o ] }
```

Comment:
Why the optional string? Isn’t the string already passed via the option(external_ref) above?
`| EXTERNAL; o = option(STRING) { assert (o = None); [ External o ] }` doesn’t break the code.
I suppose we can roll back this change.

Reply:
because I was not convinced of `symbol x : ty = "str"` that is `external "str" type x ty`.
I think we can remove it now, modulo updating the grammar in vscode and pygments.